core/packages/test/config/launch-config.toml

```
@@ -41,27 +41,67 @@ default_args = [
     name = "rococo-dave"
     validator = true
 
-# Snowbridge
+## Bridge Hub
 [[parachains]]
+id = 1013
 add_to_genesis = true
 chain_spec_path = "{{output_dir}}/spec.json"
 cumulus_based = true
-id = 1000
 
+    # run alice as parachain collator
     [[parachains.collators]]
-    command = "{{output_bin_dir}}/snowbridge"
-    name = "snowbrige-alice"
-    rpc_port = 8081
+    name = "bh-alice"
     validator = true
+    command = "{{output_bin_dir}}/polkadot-parachain"
+    rpc_port = 8081
     ws_port = 11144
     args = ["--enable-offchain-indexing=true","--pruning=archive"]
 
         [[parachains.collators.env]]
         name = "RUST_LOG"
         value = "runtime=debug,parachain=trace,cumulus-collator=trace,aura=trace,xcm=trace,ethereum-beacon-client=trace"
 
+    # run bob as parachain collator
     [[parachains.collators]]
-    command = "{{output_bin_dir}}/snowbridge"
-    name = "snowbrige-bob"
+    name = "bh-bob"
     validator = true
+    command = "{{output_bin_dir}}/polkadot-parachain"
     args = ["--enable-offchain-indexing=true","--pruning=archive"]
+
+## Statemine
+[[parachains]]
+id = 2000
+chain = "statemine-local"
+cumulus_based = true
+
+    # run alice as parachain collator
+    [[parachains.collators]]
+    name = "statemine-alice"
```

I prefer to move over to this naming:

```suggestion
    name = "statemine01"
```